rpc/src/lib.rs

```diff
@@ -149,9 +155,9 @@ impl<T> Request<T> {
 pub(crate) type PollIo<T> = Poll<Option<io::Result<T>>>;
 
 static INIT: Once = Once::new();
-static mut SEED_SPAWN: Option<Box<dyn CloneSpawn>> = None;
+static mut SEED_SPAWN: Option<Arc<Mutex<dyn Spawn>>> = None;
```

Comment:
This is the only thing I want to look into further before merging. I will do my best to investigate tonight!